src/app/models/meditation.model.js

```diff
@@ -12,6 +12,7 @@ let meditationSchema = mongoose.Schema({
   walking: Number,
   sitting: Number,
   numOfLikes: Number,
+  start: Date,
```

Comment:
We don't need that. We have `createdAt`. It's auto-generated by Mongoose, but should be editable. Please try that.